src/SUMMARY.md

```diff
@@ -2,63 +2,63 @@
 
 - [Introduction](README.md)
 - [Overview](overview.md)
-- [Service API coverage](service_api_coverage.md)
-- [Clients API coverage](clients_api_coverage.md)
```

Comment:
I removed that file because I think that Rust clients (or other language) can directly check the code documentation of their library to know what is available/missing. That avoids maintaining costs.